app/services/github_service.rb

```diff
@@ -0,0 +1,13 @@
+class GithubService
+  def conn
+    Faraday.new(url: 'https://api.github.com') do |faraday|
+      # faraday.headers['Authorization'] = ENV['GITHUB_API_KEY']
+      faraday.headers['Authorization'] = "token 484cabdee3b4e5de4226ed80470600b274e435d9"
+    end
+  end
+
+  def repos_body
+    response = conn.get('/user/repos')
+    JSON.parse(response.body, symbolize_names: true)
+  end
```

Comment:
anticipating more 'attribute'.body's

Reply:
I think this is a great method naming convention. Since the name repos (and in the future followers) will be used extensively throughout the code, calling this body is very explicit.